*** mod_alias.c~	Mon Jul  8 15:00:36 1996
--- mod_alias.c	Sun Jul 21 18:50:02 1996
***************
*** 153,158 ****
--- 153,160 ----
  command_rec alias_cmds[] = {
  { "Alias", add_alias, NULL, RSRC_CONF, TAKE2, 
      "a fakename and a realname"},
+ { "PerlAlias", add_alias, "perl-script", RSRC_CONF, TAKE2, 
+     "a fakename and a realname"},
  { "ScriptAlias", add_alias, "cgi-script", RSRC_CONF, TAKE2, 
      "a fakename and a realname"},
  { "Redirect", add_redirect, NULL, OR_FILEINFO, TAKE2, 
